src/core/ext/transport/binder/wire_format/binder_android.cc

```diff
@@ -318,6 +318,7 @@ absl::Status ReadableParcelAndroid::ReadByteArray(std::string* data) const {
   if (AParcelReadVector(parcel_, &vec) == STATUS_OK) {
     data->resize(vec.size());
     if (!vec.empty()) {
+      // TODO(mingcl): Avoid making unnecessary copy here
```

Comment:
Again, ultimately I think what we really want is (an equivalent of) directly reading into grpc_slice, so yes we can remove the unnecessary copy here, but if we want to optimize the performance we may as well just not use `std::string` completely. What do you think?

Reply:
Ah I guess this is already under that "use grpc_slice everywhere" goal then. Let me abandon this PR